webui/react/src/pages/ExperimentDetails/ExperimentTrials.table.tsx

```diff
@@ -79,6 +79,7 @@ export const columns: ColumnDef<TrialItem>[] = [
     dataIndex: 'checkpoint',
     defaultWidth: DEFAULT_COLUMN_WIDTHS['checkpoint'],
     key: 'checkpoint',
+    shouldCellUpdate: () => false,
```

Comment:
was this necessary even after the refactor to use `useCheckpointFlow`?

Reply:
Good catch, that was a failed experiment.